README.md

```diff
@@ -28,6 +28,10 @@ Make sure:
 - You have few knowledge about Html, Css, flexbox, media queries, grid, Github and Linters
 - You have installed Live Server to view the project in the browser.
 
+### VIEW DEMO
+
+https://kabika681.github.io/portfolio-pascal-kabika/
+
```

Comment:
- [ ] **Optional:** Use a link tag, this makes your readme more professional. Example; `[Live Demo](https://kabika681.github.io/portfolio-pascal-kabika/)`  => [Live Demo](https://kabika681.github.io/portfolio-pascal-kabika/)